docs/pages/architecture/proxy.mdx

```diff
@@ -11,11 +11,12 @@ provides the following key features:
   SSH and Windows desktops using the Teleport web UI.
 - Intercepts traffic for multiple protocols, including SSH, Kubernetes, HTTPS, 
   and databases, and ensures that only authenticated clients can connect to target resources.
-- Records commands, API calls, and queries and streams them to the audit log.
 - Provides networking and connectivity so that servers and proxies behind firewalls can connect
   using reverse tunnels.
 - Enables system administrators to use TLS routing feature to compress all ports for all protocols 
   to one TLS port using TLS routing feature.
+- Records commands and API calls, and queries and streams them to the audit log.
+  The audit log is stored on the Teleport Auth Service backend.
```

Comment:
I don't love the last sentence. It makes it sound like there is one backend, but the auth service has separate backends for audit log, session recordings, and cluster state.